<?xml version="1.0" encoding="UTF-8"?>
<launch>

  <node name="extrinsic_calib" pkg="image_elab" type="extrinsic_calib_node" respawn="false" output="screen"/>

  <node name="rectify_image" pkg="image_elab" type="rectify_node" respawn="true" output="screen"/>

  <node name="unwarp_image" pkg="image_elab" type="unwarp_node" respawn="true" output="screen"/>

  <node name="obstacle_detector" pkg="image_elab" type="obstacle_detector_node" respawn="true" output="screen"/>

  <node name="robot_detector" pkg="image_elab" type="robot_detector_node" respawn="true" output="screen"/>

  <node name="raw_listener" pkg="image_elab" type="raw_img_listener_node" respawn="false" output="screen"/>

  <node name="rectify_listener" pkg="image_elab" type="rectify_img_listener_node" respawn="false" output="screen"/>

  <node name="unwarp_ground_listener" pkg="image_elab" type="unwarp_ground_img_listener_node" respawn="false" output="screen"/>

  <node name="unwarp_robot_listener" pkg="image_elab" type="unwarp_robot_img_listener_node" respawn="false" output="screen"/>

</launch>

